core/chains/evm/client/errors.go

```diff
@@ -158,7 +158,7 @@ var arbitrum = ClientErrors{
 	Fatal:                 arbitrumFatal,
 	L2FeeTooLow:           regexp.MustCompile(`(: |^)max fee per gas less than block base fee(:|$)`),
 	L2Full:                regexp.MustCompile(`(: |^)(queue full|sequencer pending tx pool full, please try again)(:|$)`),
-	ServiceUnavailable:    regexp.MustCompile(`(: |^)502 Bad Gateway: [\s\S]*$`),
+	ServiceUnavailable:    regexp.MustCompile(`(: |^)502 Bad Gateway: [\s\S]*$|network is unreachable|observed invariant violation on SendTransaction`),
```

Comment:
@dhaidashenko Are these two error message enough ?

Reply:
Do we have some docs we could link for these? because `invariant violation on SendTransaction` reads like it could be considered `Fatal`, so we should link any explanation that we can.